docs/core/tutorials/cli-templates-create-project-template.md

```diff
@@ -121,6 +122,8 @@ Open the _template.json_ with your favorite text editor and paste in the followi
 
 This config file contains all of the settings for your template. You can see the basic settings such as `name` and `shortName` but also there's a `tags/type` value that's set to `project`. This designates your template as a project template. There's no restriction on the type of template you create. The `item` and `project` values are common names that .NET recommends so that users can easily filter the type of template they're searching for.
 
+The `sourceName` item is what is replaced when the user creates a new item from the template. In this case, the user will run `dotnet new stringext -n test` and the `sourceName` will be replaced with the name the user sets on item creation. The `sourceName` must match the csproj file and the namespace. In this case test will replace `ExampleTemplate.StringExtensions` in the csproj file and the namespace.
```

Comment:
Some minor corrections here:

```suggestion
The `sourceName` item is what is replaced when the user uses the template. The value of `sourceName` in the config file, is searched for in any file name and file content. In this case test will replace `consoleasync` in the name of the .csproj file. By default, this value uses the name of the current folder. When the `-n` or `--name` parameter is passed with the `dotnet new` command, the value provided is used instead of the current folder.
```